src/bundle.ts

```diff
@@ -7,88 +7,141 @@ import { pathToPointer } from './pathToPointer';
 import { pointerToPath } from './pointerToPath';
 import { traverse } from './traverse';
 
-export const BUNDLE_ROOT = '__bundled__';
-export const ERRORS_ROOT = '__errors__';
-
-export const bundleTarget = <T = unknown>({ document, path }: { document: T; path: string }, cur?: unknown) =>
-  _bundle(cloneDeep(document), path, { [path]: true }, cur);
-
-const _bundle = (
-  document: unknown,
-  path: string,
-  stack: Dictionary<boolean>,
+export const BUNDLE_ROOT = '#/__bundled__';
```

Comment:
Hmm this is a breaking change though - any reason in particular to do it?

Reply:
Support nested paths.

Reply:
We _could_ convert it within `bundleTarget` fn, though.
In any case, it felt weird that we accept a pointer in one case, and not the other.